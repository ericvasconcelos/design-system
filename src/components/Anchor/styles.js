import styled from 'styled-components'
import { getButtonStyle } from 'utils/getButtonStyle'

export const Wrapper = styled.a`
  ${getButtonStyle}
`
