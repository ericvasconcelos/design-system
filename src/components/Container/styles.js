import styled from 'styled-components'

export const Wrapper = styled.div`
  margin: auto;
  width: ${({ theme }) => theme.grid.smMobile[1]};

  @media (min-width: 576px) {
    width: ${({ theme }) => theme.grid.mobile[1]};
  }

  @media (min-width: 768px) {
    width: ${({ theme }) => theme.grid.smTablet[1]};
  }

  @media (min-width: 992px) {
    width: ${({ theme }) => theme.grid.tablet[1]};
  }

  @media (min-width: 1200px) {
    width: ${({ theme }) => theme.grid.laptop[1]};
  }

  @media (min-width: 1400px) {
    width: ${({ theme }) => theme.grid.desktop[1]};
  }
`
