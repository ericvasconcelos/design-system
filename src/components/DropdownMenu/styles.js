import styled from 'styled-components'
import { getColorFromName } from 'utils/getColorFromName'

export const Wrapper = styled.ul`
  background-color: ${({ theme }) => theme.pallete.greyCash[100]};
  border: 1px solid ${({ theme }) => theme.pallete.greyCash[300]};
  padding: ${({ theme }) => theme.padding.sm};
  border-radius: 1rem;
  box-shadow: 0px 8px 16px 0px
    ${({ theme }) => getColorFromName({ theme, styleColor: 'greyCash500-08' })};
  list-style: none;
  display: flex;
  flex-direction: column;
  gap: ${({ theme }) => theme.gutters.sm};
  align-items: stretch;
`
