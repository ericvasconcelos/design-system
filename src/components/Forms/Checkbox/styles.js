import styled from 'styled-components'
import { getColorFromName } from 'utils/getColorFromName'

export const Label = styled.label`
  display: flex;
  align-items: center;
`

export const StyledInput = styled.input`
  appearance: none;

  background-color: ${({ theme, disabled }) =>
    disabled
      ? getColorFromName({ theme, styleColor: 'greyCash400-08' })
      : theme.pallete.greyCash[100]};
  border: 2px solid
    ${({ theme, disabled }) =>
    disabled
      ? getColorFromName({ theme, styleColor: 'greyCash400-08' })
      : theme.pallete.greyCash[300]};
  border-radius: ${({ theme }) => theme.border.radius.xs};
  display: inline-block;
  position: relative;
  padding: 10px;
  margin-right: ${({ theme }) => theme.gutters.md};

  &[indeterminate] {
    background-color: ${({ theme }) => theme.pallete.success[400]};
    border-color: ${({ theme }) => theme.pallete.success[400]};

    &:after {
      content: '';
      width: 8px;
      height: 2px;
      background-color: ${({ theme }) => theme.pallete.greyCash[100]};
      position: absolute;
      top: 9px;
      left: 6px;
      border-radius: ${({ theme }) => theme.border.radius.xs};
    }
  }

  &:not([disabled]):hover,
  &:not([disabled]):focus {
    border-color: ${({ theme }) => theme.pallete.lightBlueCash[300]};
    box-shadow: 0 0 0 4px
      ${({ theme }) => getColorFromName({ theme, styleColor: 'lightBlueCash300-24' })};
  }

  &[checked] {
    ${({ disabled, theme }) =>
    !disabled &&
      `
        background-color: ${theme.pallete.success[400]};
        border-color: ${theme.pallete.success[400]};
      `}

    &:after {
      content: ' ';
      mask-image: url(${({ theme }) => theme.icons.checkBold});
      mask-size: contain;
      background-color: ${({ theme, disabled }) =>
    disabled
      ? getColorFromName({ theme, styleColor: 'greyCash400-32' })
      : theme.pallete.greyCash[100]};
      width: 8px;
      height: 6px;
      position: absolute;
      top: 7px;
      left: 6px;
    }
  }
`
