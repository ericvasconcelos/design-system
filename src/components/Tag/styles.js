import styled from 'styled-components'
import { getColorFromName } from 'utils/getColorFromName'
import { getTypography } from 'utils/getTypography'

const getColor = ({ styleType, theme }) => {
  const colors = {
    default: theme.pallete.lightBlueCash[500],
    darkBlue: theme.pallete.greyCash[100],
    lightBlue: theme.pallete.lightBlueCash[300],
    cyan: theme.pallete.cyanCash[500],
    gold: theme.pallete.goldCash[400],
    gray: theme.pallete.darkBlueCash[200],
    warning: theme.pallete.warning[400],
    danger: theme.pallete.error[300],
    success: theme.pallete.success[500]
  }

  return colors[styleType]
}

const getBackgroundColor = ({ styleType, theme }) => {
  const colors = {
    default: getColorFromName({ theme, styleColor: 'lightBlueCash200-40' }),
    darkBlue: theme.pallete.darkBlueCash[300],
    gold: theme.pallete.goldCash[100],
    cyan: theme.pallete.cyanCash[200],
    gray: getColorFromName({ theme, styleColor: 'darkBlueCash200-16' }),
    lightBlue: theme.pallete.lightBlueCash[100],
    warning: getColorFromName({ theme, styleColor: 'warning400-16' }),
    danger: getColorFromName({ theme, styleColor: 'error300-16' }),
    success: getColorFromName({ theme, styleColor: 'success500-16' })
  }

  return colors[styleType]
}

export const Wrapper = styled.div`
  display: inline-block;
  ${({ theme }) => getTypography({ theme, styleType: 'tag' })}
  text-transform: uppercase;
  padding: ${({ theme }) => `0 ${theme.padding.sm}`};
  border-radius: ${({ theme }) => theme.border.radius.sm};
  color: ${getColor};
  background-color: ${getBackgroundColor};

  > span {
    display: inline-block;
  }
`
