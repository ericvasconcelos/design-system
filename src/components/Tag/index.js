import React from 'react'

import { oneOf, oneOfType, string, node, element, elementType } from 'prop-types'

import { Wrapper } from './styles'

export const Tag = ({ styleType, text, children, className }) => (
  <Wrapper data-testid="Tag" styleType={styleType} className={className}>
    <span>{text}</span>
    {children}
  </Wrapper>
)

Tag.defaultProps = {
  className: '',
  children: '',
  styleType: 'default'
}

Tag.propTypes = {
  /** className permite estilização em outros lugares */
  className: string,
  /** styleType define a cor de uso dentro das cores abaixo */
  styleType: oneOf([
    'lightBlue',
    'darkBlue',
    'cyan',
    'gold',
    'gray',
    'success',
    'warning',
    'danger',
    'default'
  ]),
  /** Children elementos que vão dentro da Tag além do texto. */
  children: oneOfType([node, element, elementType, string]),
  /** Text define o texto escrito no label */
  text: string.isRequired
}
