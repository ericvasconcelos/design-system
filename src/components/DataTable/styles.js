import { Icon } from 'components/Icon'
import { Loader as LoaderComponent } from 'components/Loader'
import { Text as TextComponent } from 'components/Text'
import styled from 'styled-components'
import { getColorFromName } from 'utils/getColorFromName'
import { getTypography } from 'utils/getTypography'

export const WrapperTable = styled.div`
  @media (max-width: 1024px) {
    overflow-x: auto;
  }
`

export const Loader = styled(LoaderComponent)`
  margin: auto;
  margin-top: unset;
  height: 100%;
`

export const Table = styled.table`
  border-collapse: separate;
  width: 100%;
  margin: auto;
  border-spacing: ${({ theme }) => `0 ${theme.gutters.sm}`};

  td {
    ${({ theme }) =>
    getTypography({
      theme,
      styleType: 'body2'
    })}
  }

  td:first-child,
  th:first-child {
    border-radius: 8px 0 0 8px;
  }

  td:last-child,
  th:last-child {
    border-radius: 0 8px 8px 0;
  }
`

export const Header = styled.th`
  padding: ${({ theme }) => theme.padding.md};
  position: relative;
  cursor: ${({ sortable }) => (sortable ? 'pointer' : 'auto')};
`

export const HeaderContent = styled.div`
  display: flex;
  align-items: center;
  gap: ${({ theme }) => theme.gutters.sm};
  white-space: nowrap;
  text-align: left;
  ${({ theme }) =>
    getTypography({
      theme,
      styleType: 'body2Medium',
      styleColor: 'darkBlueCash500'
    })}
`

export const IconSort = styled(Icon)`
  width: 16px;
  height: 16px;
  mask-size: 16px;
  background: ${({ theme }) => theme.pallete.darkBlueCash[500]};
`

export const Column = styled.th`
  padding: ${({ theme }) => theme.padding.md};
  text-align: left;
  border: 1px solid transparent;
  &:focus {
    border: 1px solid ${({ theme }) => theme.pallete.lightBlueCash[200]};
    box-shadow: 0 0 0 4px
      ${({ theme }) => getColorFromName({ theme, styleColor: 'lightBlueCash300-24' })};
  }
  ${({ theme }) =>
    getTypography({
      theme,
      styleType: 'body2',
      styleColor: 'darkBlueCash500'
    })}
`

export const ClickableColumn = styled(Column)`
  cursor: pointer;
  padding: 0;
  vertical-align: middle;
  line-height: 0;
  max-width: 80px;

  
  
  `

export const Row = styled.tr`
  border-bottom: 1px solid rgba(0, 0, 0, 0.08);
  display: ${({ collapse }) => (collapse ? 'none' : 'table-row')};
  background-color: ${({ theme }) => theme.pallete.greyCash[100]};
  border-top-left-radius: 10px;
  
  

`

export const Pagination = styled.div`
  text-align: center;
`

export const ShowMoreButton = styled.button`
  display: flex;
  justify-content: center;
  &:focus {
    border: 1px solid ${({ theme }) => theme.pallete.lightBlueCash[200]};
    box-shadow: 0 0 0 4px
      ${({ theme }) => getColorFromName({ theme, styleColor: 'lightBlueCash300-24' })};
  }
  ${({ theme }) =>
    getTypography({
      theme,
      styleType: 'body2Medium',
      styleColor: 'lightBlueCash300'
    })}
  cursor: ${({ disabled }) => (disabled ? 'not-allowed' : 'pointer')};
  background-color: ${({ theme }) => theme.pallete.greyCash[100]};
  width: 100%;
  border-radius: 8px;
  padding: ${({ theme }) => theme.padding.lg};
`

export const Text = styled(TextComponent)`
  display: flex;
  justify-content: center;
  ${({ theme }) =>
    getTypography({
      theme,
      styleType: 'body2',
      styleColor: 'darkBlueCash500'
    })}
  cursor: ${({ disabled }) => (disabled ? 'not-allowed' : 'auto')};
  background-color: ${({ theme }) => theme.pallete.greyCash[100]};
  width: 100%;
  border-radius: 8px;
  padding: ${({ theme }) => theme.padding.lg};
`

export const ButtonIcon = styled.button`
  cursor: pointer;
`
