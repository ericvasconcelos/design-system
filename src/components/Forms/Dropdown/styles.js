import { Button as BaseButton } from 'components/Button'
import { Icon } from 'components/Icon'
import styled from 'styled-components'
import { getColorFromName } from 'utils/getColorFromName'
import { getTypography } from 'utils/getTypography'

import { BaseIconWrapper, BaseInput, BaseInputWrapper, BaseLabel } from '../commonStyles'

export const IconWrapper = styled(BaseIconWrapper)`
  grid-area: icon;
`

export const OptionIcon = styled(Icon)``

export const CustomSelect = styled(BaseInputWrapper)`
  display: grid;
  grid-template: 'label icon' 1.0625rem 'input icon' 1.375rem / minmax(0, 1fr) 1.5rem;
  column-gap: 1rem;
  row-gap: 0.1875rem;
  position: relative;
  white-space: nowrap;
`

export const CustomSelectCheckbox = styled(CustomSelect)`
  cursor: pointer;
`

export const Label = styled(BaseLabel)`
  grid-area: label;
`

export const DisplaySelection = styled(BaseInput)`
  grid-area: input;
  overflow: hidden;
  text-overflow: ellipsis;
  color: ${({ value, theme }) =>
    value === '' ? theme.pallete.darkBlueCash[200] : theme.pallete.darkBlueCash[500]};
`

export const OptionsBox = styled.ul`
  background-color: ${({ theme }) => theme.pallete.greyCash[100]};
  border-radius: 0.5rem;
  border: 1px solid;
  border-color: ${({ theme }) => theme.pallete.greyCash[300]};
  height: calc(
    ${({ numberOptions, optionSize }) =>
    `${optionSize === 'large' ? '3.5rem' : '2.5rem'} * ${
      numberOptions > 5 ? 5 : numberOptions
    } + 1rem`}
  );
  width: 100%;
  box-sizing: border-box;
  z-index: 10;
  overflow-y: auto;
  display: ${({ active }) => (active ? 'block' : 'none')};
  padding: ${({ theme }) => theme.padding.sm};
`

const BaseOption = styled.li`
  display: flex;
  align-items: center;
  ${({ theme }) => getTypography({ theme, styleType: 'input', styleColor: 'darkBlueCash500' })};
  padding: ${({ theme }) => theme.padding.md};
  gap: ${({ theme }) => theme.padding.sm};
  line-height: 1;
  border-radius: 0.25rem;
  pointer-events: auto;
  height: ${({ optionSize }) => (optionSize === 'large' ? '3.5rem' : '2.5rem')};
`

export const Option = styled(BaseOption)`
  cursor: pointer;
  background-color: ${({ theme, hovered }) =>
    hovered ? getColorFromName({ theme, styleColor: 'lightBlueCash300-24' }) : 'unset'};

  :hover {
    background-color: ${({ theme }) =>
    getColorFromName({ theme, styleColor: 'lightBlueCash100' })};
    color: ${({ theme }) => theme.pallete.lightBlueCash[300]};

    ${OptionIcon} {
      background-color: ${({ theme }) => theme.pallete.lightBlueCash[100]};
    }
  }

  :focus {
    background-color: ${({ theme }) =>
    getColorFromName({ theme, styleColor: 'lightBlueCash100' })};
    color: ${({ theme }) => theme.pallete.lightBlueCash[300]};

    ${OptionIcon} {
      background-color: ${({ theme }) => theme.pallete.lightBlueCash[300]};
    }
  }
`

export const OptionCheckbox = styled(BaseOption)`
  label,
  input {
    cursor: pointer;
  }
`

export const HiddenSelect = styled.select`
  visibility: hidden;
  position: absolute;
  top: 0;
  left: 0;
`

export const CheckboxOptionsWrapper = styled.div`
  display: ${({ active }) => (active ? 'block' : 'none')};
  background-color: ${({ theme }) => theme.pallete.greyCash[100]};
  position: relative;
  height: calc(
    ${({ numberOptions, optionSize }) =>
    `${optionSize === 'large' ? '3.5rem' : '2.5rem'} * ${
      numberOptions > 4 ? 4 : numberOptions
    } + 1rem`}
  );
  border-radius: 0.5rem;
  border: 1px solid;
  border-color: ${({ theme }) => theme.pallete.greyCash[300]};
`

export const CheckboxOptionsBox = styled.ul`
  height: calc(
    ${({ numberOptions, optionSize }) =>
    `${optionSize === 'large' ? '3.5rem' : '2.5rem'} * ${
      numberOptions > 3 ? 3 : numberOptions
    } + 1rem`}
  );
  border-radius: 0.5rem;
  width: 100%;
  box-sizing: border-box;
  z-index: 10;
  overflow-y: auto;
  padding: ${({ theme }) => theme.padding.sm};
`

export const ClearOptions = styled.div`
  display: inline-flex;
  width: 100%;
  justify-content: center;
  align-items: center;
  border-top: 1px solid ${({ theme }) => theme.pallete.greyCash[300]};
  border-radius: 0rem 0rem 0.5rem 0.5rem;
  ${({ theme }) => getTypography({ theme, styleType: 'body2', styleColor: 'lightBlueCash300' })};
  height: ${({ optionSize }) => (optionSize === 'large' ? '3.5rem' : '2.5rem')};
`

export const Button = styled(BaseButton)`
  text-transform: unset;
  height: 100%;
  font-weight: 500 !important;
  width: auto;
`
