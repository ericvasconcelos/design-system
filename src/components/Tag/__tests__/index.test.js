import React from 'react'

import { screen } from '@testing-library/react'
import { Tag } from 'components/Tag'
import { theme } from 'styles/theme'
import { getColorFromName } from 'utils/getColorFromName'
import { styledRender } from 'utils/styledRender'

describe('Render Tag Component when', () => {
  it('styleType no style prop', () => {
    const { container } = styledRender(<Tag text="Tag" />)
    const label = screen.getByTestId('Tag')

    expect(label).toBeInTheDocument()
    expect(label).toHaveStyle({
      color: theme.pallete.lightBlueCash[500],
      backgroundColor: getColorFromName({ theme, styleColor: 'lightBlueCash200-40' })
    })
    expect(container.firstChild).toMatchSnapshot()
  })

  it('styleType is default', () => {
    const { container } = styledRender(<Tag styleType="default" text="Tag" />)
    const label = screen.getByTestId('Tag')

    expect(label).toBeInTheDocument()
    expect(label).toHaveStyle({
      color: theme.pallete.lightBlueCash[500],
      backgroundColor: getColorFromName({ theme, styleColor: 'lightBlueCash200-40' })
    })
    expect(container.firstChild).toMatchSnapshot()
  })

  it('styleType is darkBlue', () => {
    const { container } = styledRender(<Tag styleType="darkBlue" text="Tag" />)
    const label = screen.getByTestId('Tag')

    expect(label).toBeInTheDocument()
    expect(label).toHaveStyle({
      color: theme.pallete.greyCash[100],
      backgroundColor: theme.pallete.darkBlueCash[300]
    })
    expect(container.firstChild).toMatchSnapshot()
  })

  it('styleType is lightBlue', () => {
    const { container } = styledRender(<Tag styleType="lightBlue" text="Tag" />)
    const label = screen.getByTestId('Tag')

    expect(label).toBeInTheDocument()
    expect(label).toHaveStyle({
      color: theme.pallete.lightBlueCash[300],
      backgroundColor: theme.pallete.lightBlueCash[100]
    })

    expect(container.firstChild).toMatchSnapshot()
  })

  it('styleType is gold', () => {
    const { container } = styledRender(<Tag styleType="gold" text="Tag" />)
    const label = screen.getByTestId('Tag')

    expect(label).toBeInTheDocument()
    expect(label).toHaveStyle({
      color: theme.pallete.goldCash[400],
      backgroundColor: theme.pallete.goldCash[100]
    })

    expect(container.firstChild).toMatchSnapshot()
  })

  it('styleType is gray', () => {
    const { container } = styledRender(<Tag styleType="gray" text="Tag" />)
    const label = screen.getByTestId('Tag')

    expect(label).toBeInTheDocument()
    expect(label).toHaveStyle({
      color: theme.pallete.darkBlueCash[200],
      backgroundColor: getColorFromName({ theme, styleColor: 'darkBlueCash200-16' })
    })

    expect(container.firstChild).toMatchSnapshot()
  })

  it('styleType is warning', () => {
    const { container } = styledRender(<Tag styleType="warning" text="Tag" />)
    const label = screen.getByTestId('Tag')

    expect(label).toBeInTheDocument()
    expect(label).toHaveStyle({
      color: theme.pallete.warning[400],
      backgroundColor: getColorFromName({ theme, styleColor: 'warning400-16' })
    })
    expect(container.firstChild).toMatchSnapshot()
  })

  it('styleType is danger', () => {
    const { container } = styledRender(<Tag styleType="danger" text="Tag" />)
    const label = screen.getByTestId('Tag')

    expect(label).toBeInTheDocument()
    expect(label).toHaveStyle({
      color: theme.pallete.error[300],
      backgroundColor: getColorFromName({ theme, styleColor: 'error300-16' })
    })
    expect(container.firstChild).toMatchSnapshot()
  })
})
