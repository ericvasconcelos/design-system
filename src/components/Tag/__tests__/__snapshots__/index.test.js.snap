// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render Tag Component when styleType is danger 1`] = `
.c0 {
  display: inline-block;
  color: none;
  font-size: 0.5rem;
  font-weight: 500;
  line-height: 1rem;
  -webkit-text-decoration: none;
  text-decoration: none;
  -webkit-letter-spacing: 0rem;
  -moz-letter-spacing: 0rem;
  -ms-letter-spacing: 0rem;
  letter-spacing: 0rem;
  font-family: 'DM Sans',sans-serif;
  text-transform: uppercase;
  padding: 0 0.5rem;
  border-radius: 0.5rem;
  color: #FF4949;
  background-color: #FF494929;
}

.c0 > span {
  display: inline-block;
}

<div
  class="c0"
  data-testid="Tag"
>
  <span>
    Tag
  </span>
  
</div>
`;

exports[`Render Tag Component when styleType is darkBlue 1`] = `
.c0 {
  display: inline-block;
  color: none;
  font-size: 0.5rem;
  font-weight: 500;
  line-height: 1rem;
  -webkit-text-decoration: none;
  text-decoration: none;
  -webkit-letter-spacing: 0rem;
  -moz-letter-spacing: 0rem;
  -ms-letter-spacing: 0rem;
  letter-spacing: 0rem;
  font-family: 'DM Sans',sans-serif;
  text-transform: uppercase;
  padding: 0 0.5rem;
  border-radius: 0.5rem;
  color: #FFFFFF;
  background-color: #00224B;
}

.c0 > span {
  display: inline-block;
}

<div
  class="c0"
  data-testid="Tag"
>
  <span>
    Tag
  </span>
  
</div>
`;

exports[`Render Tag Component when styleType is default 1`] = `
.c0 {
  display: inline-block;
  color: none;
  font-size: 0.5rem;
  font-weight: 500;
  line-height: 1rem;
  -webkit-text-decoration: none;
  text-decoration: none;
  -webkit-letter-spacing: 0rem;
  -moz-letter-spacing: 0rem;
  -ms-letter-spacing: 0rem;
  letter-spacing: 0rem;
  font-family: 'DM Sans',sans-serif;
  text-transform: uppercase;
  padding: 0 0.5rem;
  border-radius: 0.5rem;
  color: #02279E;
  background-color: #8DC6FF66;
}

.c0 > span {
  display: inline-block;
}

<div
  class="c0"
  data-testid="Tag"
>
  <span>
    Tag
  </span>
  
</div>
`;

exports[`Render Tag Component when styleType is gold 1`] = `
.c0 {
  display: inline-block;
  color: none;
  font-size: 0.5rem;
  font-weight: 500;
  line-height: 1rem;
  -webkit-text-decoration: none;
  text-decoration: none;
  -webkit-letter-spacing: 0rem;
  -moz-letter-spacing: 0rem;
  -ms-letter-spacing: 0rem;
  letter-spacing: 0rem;
  font-family: 'DM Sans',sans-serif;
  text-transform: uppercase;
  padding: 0 0.5rem;
  border-radius: 0.5rem;
  color: #B37533;
  background-color: #F7EBD2;
}

.c0 > span {
  display: inline-block;
}

<div
  class="c0"
  data-testid="Tag"
>
  <span>
    Tag
  </span>
  
</div>
`;

exports[`Render Tag Component when styleType is gray 1`] = `
.c0 {
  display: inline-block;
  color: none;
  font-size: 0.5rem;
  font-weight: 500;
  line-height: 1rem;
  -webkit-text-decoration: none;
  text-decoration: none;
  -webkit-letter-spacing: 0rem;
  -moz-letter-spacing: 0rem;
  -ms-letter-spacing: 0rem;
  letter-spacing: 0rem;
  font-family: 'DM Sans',sans-serif;
  text-transform: uppercase;
  padding: 0 0.5rem;
  border-radius: 0.5rem;
  color: #455A73;
  background-color: #455A7329;
}

.c0 > span {
  display: inline-block;
}

<div
  class="c0"
  data-testid="Tag"
>
  <span>
    Tag
  </span>
  
</div>
`;

exports[`Render Tag Component when styleType is lightBlue 1`] = `
.c0 {
  display: inline-block;
  color: none;
  font-size: 0.5rem;
  font-weight: 500;
  line-height: 1rem;
  -webkit-text-decoration: none;
  text-decoration: none;
  -webkit-letter-spacing: 0rem;
  -moz-letter-spacing: 0rem;
  -ms-letter-spacing: 0rem;
  letter-spacing: 0rem;
  font-family: 'DM Sans',sans-serif;
  text-transform: uppercase;
  padding: 0 0.5rem;
  border-radius: 0.5rem;
  color: #003FF2;
  background-color: #D7EBFF;
}

.c0 > span {
  display: inline-block;
}

<div
  class="c0"
  data-testid="Tag"
>
  <span>
    Tag
  </span>
  
</div>
`;

exports[`Render Tag Component when styleType is warning 1`] = `
.c0 {
  display: inline-block;
  color: none;
  font-size: 0.5rem;
  font-weight: 500;
  line-height: 1rem;
  -webkit-text-decoration: none;
  text-decoration: none;
  -webkit-letter-spacing: 0rem;
  -moz-letter-spacing: 0rem;
  -ms-letter-spacing: 0rem;
  letter-spacing: 0rem;
  font-family: 'DM Sans',sans-serif;
  text-transform: uppercase;
  padding: 0 0.5rem;
  border-radius: 0.5rem;
  color: #FF7935;
  background-color: #FF793529;
}

.c0 > span {
  display: inline-block;
}

<div
  class="c0"
  data-testid="Tag"
>
  <span>
    Tag
  </span>
  
</div>
`;

exports[`Render Tag Component when styleType no style prop 1`] = `
.c0 {
  display: inline-block;
  color: none;
  font-size: 0.5rem;
  font-weight: 500;
  line-height: 1rem;
  -webkit-text-decoration: none;
  text-decoration: none;
  -webkit-letter-spacing: 0rem;
  -moz-letter-spacing: 0rem;
  -ms-letter-spacing: 0rem;
  letter-spacing: 0rem;
  font-family: 'DM Sans',sans-serif;
  text-transform: uppercase;
  padding: 0 0.5rem;
  border-radius: 0.5rem;
  color: #02279E;
  background-color: #8DC6FF66;
}

.c0 > span {
  display: inline-block;
}

<div
  class="c0"
  data-testid="Tag"
>
  <span>
    Tag
  </span>
  
</div>
`;
